Examples/MAX32655/RF_Test/CLI-commands.c

```diff
@@ -489,14 +493,23 @@ static BaseType_t cmd_ConstTx(char *pcWriteBuffer, size_t xWriteBufferLen,
         snprintf(pcWriteBuffer, (size_t)MAX_OUTPUT_STRING_SIZE, "> Starting constant TX\r\n");
         PalBbEnable();
         llc_api_tx_ldo_setup();
+// TODO delete debug comments before merge
+//-------------------------- const tx stuff ----------------
+        if(testVersion){
+            PalBbBleEnablePrbs15(TRUE);
+                /* 0 num of packets for infinite*/
+            LlTxTest(channelNum , 255 ,LL_TEST_PKT_TYPE_PRBS15, 0);
+                /* Enable constant TX */
+        }
+        else{
+            MXC_R_TX_CTRL = 0x1;
 
-        /* Enable constant TX */
-        MXC_R_TX_CTRL = 0x1;
-
-        /* Enable pattern generator, set PRBS-9 */
-        MXC_R_CONST_OUPUT = 0x0;
-        MXC_R_PATTERN_GEN = 0x4B;
+            /* Enable pattern generator, set PRBS-9 */
+            MXC_R_CONST_OUPUT = 0x0;
+            MXC_R_PATTERN_GEN = 0x4B;
+        }
```

Comment:
optionally running 2 versions of the test for debugging.
Using PalBbBleEnablePrbs15 generates the large waveforms in Universal Radio Hacker (URH)
and the smaller waveforms are generated to the way it was being done originally. 
I thought it was a power discrepency so I changed power to 4dBm but still the new version is significantly different.
I gather one version is right or better, than the other.